multiaccounts/settings/database.go

```diff
@@ -144,6 +145,14 @@ INSERT INTO settings (
 		return err
 	}
 
+	if s.DisplayName != "" {
+		now := time.Now().Unix()
+		err = db.SetSettingLastSynced(DisplayName, uint64(now))
```

Comment:
could u explain why we need call `db.SetSettingLastSynced` here, as i saw there'r already some places take care of settings_sync_clock ? @saledjenic

Reply:
@qfrank yes, sure, will try to explain it now, in general it covers a special case.

Starting from an empty state (empty local data folder, no database) regardless which flow you follow to create an account (is it create a new account, create an account using seed phrase or recover account or whatever) initial settings will be inserted into `settings` table using `CreateSettings` where client client sent them via settings json object. That means any value we want to track a sync time of will be set in the settings table without any proper time set in the `settings_sync_clock`. Since other values have a default value will continue explanation on display name, cause empty string `""` is the default name for it. Display name is set by user during the onboarding flow, and if we leave the code without this check then `0` will be set for display name in `settings_sync_clock` table.

So when we perform a backup, the time of last change of display name will be `0`. Then if user recovers an account using the same seed phrase fetched display name won't be applied cause its time in backed up message will be `0` which is the same time as for display name stored in `settings_sync_clock` for an empty display name (cause during recover account flow there is no option to set display name, but it's expected that it will be fetched form waku) so user gets landed in the app without set display name in that case. To overcome that it's enough to set initial time to anything greater than `0`, initially I set it to `1` and that works, but @cammellos suggested to use current time here.

My opinion is that we need to set last sync time for all fields we want to track a last sync time of, not only for display name. That should be set to current time or last clock in case of all but the recover onboarding flows. But why it's not done that way, cause that firstly need to be discussed and secondly I was trying to apply this feature with as few possible changes as possible.